explainer.md

```diff
@@ -102,6 +102,10 @@ When the user uninstalls this web application, registered file handlers and icon
 
 For more advanced use cases, such as opening a file in an existing window or displaying a notification, applications can add a [launch event handler](https://github.com/WICG/sw-launch/blob/master/explainer.md).
 
+## Permissions API Integration
+
+A `"file-handling"` permission will be defined, in order to protect the user from accidentally allowing a PWA to view files. This permission prompt may asynchronously show right after the user chooses to use a PWA capabile of handling the associated file type, to open a file, so that the permission context is more understandable and relevant. This permission prompt should also show after the PWA has an opportunity to load, and the prompt should clearly show the PWA origin, so that it is clear to the user the identity of the PWA requesting file read access for the handled file. Launched files should only be queued into the `launchQueue` after the user accepts the permission and allows the PWA to view the relevant file types. User agents may retain control over default settings and how (or if) they are exposed to the user.
```

Comment:
Would recommend breaking this line up into 80 chars / line to make adding review comments easier.

s/capabile/capable/

Recommend breaking up the sentence: This permission prompt should also show after the PWA has ... to become two separate sentences like:

This permission prompt should also show after the PWA has an opportunity to load.
The prompt should clearly show the PWA origin, so that it is clear to the user the identity of the PWA requesting file read access for the handled file.

Reply:
This file generally doesn't use 80 char/line, so I've been following this convention in my PRs.

Thanks. I've updated the PR to match the rest of these comments.

Reply:
To clarify, I think you should break the current file convention and convert the file to 80 chars/line as edits come in.  However, please LMK if there's some benefit to not following 80 chars/line other than trying to match the rest of the file.

Reply:
Fair. I'll try using to 80 char/line in future diffs. I think I've just tended to not do so in Github since most explainers I've seen in the past tended not to, but I see that 80 char/line is used for explainers as well (like https://github.com/WICG/file-system-access/edit/main/EXPLAINER.md)